### ISSUE TITLE
Search Modes documentation

### ISSUE THREAD

**Issue by robbykrlos** [laravel-enso/filters#3]
This is  a **question**.

Hi,

We'd like to update the documentation on how the table search works, due to some research we've made, and lack of documentation on this matter (there is this : https://docs.laravel-enso.com/frontend/search-mode.html but does not explain the table modes). 

Sorry, maybe for you this seems weird, but for us was the first time encountering this kind of search and was not easy to understand all the modes, and how the modes are displayed...

We have created the documentation, describing search modes inside table search,  with examples

I see that the SearchModes (code) are described inside the filters module on BE, but, used on FE in multiple core places (one of which is "CoreSearch" from tables modules. Of course the FE has also the SearchMode module.

I would be tempted to update https://docs.laravel-enso.com/frontend/search-mode.html but does not explain the table modes with some description on modes but also extend the table documentation with this extra search chapter. Let me know if you agree.

We'll create a PR afterwards.

**Comment by aocneanu** [comment]
@robbykrlos thank you for this initiative.

@gandesc please share your opinion before we proceed.

**Comment by gandesc** [comment]
Hi @robbykrlos 

We've tried documenting features where we thought it made sense from our perspective, but someone with a fresh perspective
might see things in a different way, so I welcome your PR.

**Comment by robbykrlos** [comment]
@gandesc will do 👍

**Comment by robbykrlos** [comment]
@gandesc - PR done - https://github.com/laravel-enso/documentation/pull/50.

As you will see, we tried to point our some things that weren't that obvious for us, and added some additional specific mentions for some specific situations that again, were not that natural. 

We spent like ~2 h trying to reformulate and integrate our documentation with your, and came to the conclusion that maybe we made too much of a big deal out of this. But in the end, feel free to reformulate or correct our changes further more. If some ideas are kept in, it's fine by us.

Thanks for your support.
